lib/client-assets.php

```diff
@@ -544,6 +544,11 @@ function gutenberg_register_scripts_and_styles() {
 		filemtime( gutenberg_dir_path() . 'build/format-library/index.js' ),
 		true
 	);
+	wp_add_inline_script(
+		'wp-format-library',
+		'wp.formatLibrary.registerCoreFormatTypes();',
+		'after'
+	);
```

Comment:
How can we make sure someone use the `edit-post` module (via npm) get the formats by default? 

Can we remove the `wp_enqueue_script` (and style) in favor of an explicit dependency towards  `wp-format-library` in `wp-edit-post`?

Reply:
Does it mean, we would call `registerCoreFormatTypes` in `edit-post` instead?

Reply:
Maybe, It's not perfect because people that want to unregister these styles would need `edit-post` as a dependency. I'd love other opinions.